ShopLens/VoicedText/TextVoicers/ITextVoicer.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace VoicedText.TextVoicers
+{
+    /// <summary>
+    /// Interface that defines methods needed for a text voicer.
+    /// </summary>
+    public interface ITextVoicer
+    {
+        /// <summary>
+        /// Sets the speed of the text voicer.
+        /// </summary>
+        /// <returns></returns>
+        void SetSpeed(int speedOfVoicer);
+
+        /// <summary>
+        /// Sets the volume of the text voicer.
+        /// </summary>
+        /// /// <returns></returns>
+        void SetVolume(int volumeOfVoicer);
```

Comment:
Like my previous commment.

Here it's even more confusing - do you set it in scale 0-100 (percent)? Or in 1-2-3-4 (levels)?

Should also expose enum if we're doing levels.

Reply:
Then I won't have to cast those enum values to int, right? I didn't even think of this!